### ISSUE TITLE
[goblin char] finds Goblin Chariot but not Goblin Charbelcher

### ISSUE THREAD

**Issue by bakert** [PennyDreadfulMTG/Penny-Dreadful-Tools#209]

**Comment by bakert** [comment]
```
sqlite> SELECT distance, word FROM fuzzy WHERE word MATCH '*Goblin Char*';      
50|Goblin Chariot
50|Goblin Charbelcher
90|Goblin Sharpshooter
90|Goblin Chirurgeon
91|Goblin Gardener
110|Goblin Shrine
111|Goblin Grappler
111|Goblin Grenadiers
111|Goblin Grenade
112|Goblin Cohort
126|Goblin Warchief Avatar
126|Goblin War Buggy
126|Goblin War Cry
126|Goblin War Wagon
126|Goblin Cannon
126|Goblin War Paint
126|Goblin Wardriver
126|Goblin War Drums
126|Goblin Warchief
126|Goblin Warrens
```

So not sure what is going on.

**Comment by bakert** [comment]
Indentation error. Fixed by f9df134.
